lib/iris/tests/test_analysis.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2014, Met Office
+# (C) British Crown Copyright 2010 - 2015, Met Office
```

Comment:
This should be changed back to 2014 since you reverted the changes to this file.